src/Eloquent/MassPrunable.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace MongoDB\Laravel\Eloquent;
+
+use Illuminate\Database\Eloquent\MassPrunable as EloquentMassPrunable;
+use Illuminate\Database\Events\ModelsPruned;
+
+trait MassPrunable
+{
+    use EloquentMassPrunable;
+
+    /**
+     * Prune all prunable models in the database.
```

Comment:
This wording comes Frome Laravel, but I don't like that model=document in their terminology. 
```suggestion
     * Prune all prunable documents in the database.
```

Reply:
In my opinion models would be good choice because we all Laravel dev are stick with that, when we think about document where should our documents be stored? It's still under the `models/` directory right?